modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/src/main/resources/content/Language.properties

```diff
@@ -1,4 +1,2 @@
-#if (${liferayVersion.startsWith("7.1")})
-${artifactId}-description=${artifactId} Description
-#end
+#if (${liferayVersion.startsWith("7.1")})${artifactId}-description=${artifactId} Description#end
```

Comment:
I think the #end can be on the next line right?  Or does it print an extra newline for 7.0 projects?

Reply:
I'm not sure, @gamerson. I've needed to do that because a test was complaining about the first way I've added the `if`, please see the test which failed by that time:

```
com.liferay.project.templates.ProjectTemplateFilesTest > testProjectTemplateFiles FAILED
    java.lang.AssertionError: Incorrect line "#if (${liferayVersion.startsWith("7.1")})" in ../project-templates-form-field/src/main/resources/archetype-resources/src/main/resources/content/Language.properties. Actual: -1
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failEquals(Assert.java:185)
        at org.junit.Assert.assertNotEquals(Assert.java:199)
        at com.liferay.project.templates.ProjectTemplateFilesTest._testLanguageProperties(ProjectTemplateFilesTest.java:515)
        at com.liferay.project.templates.ProjectTemplateFilesTest.access$000(ProjectTemplateFilesTest.java:69)
        at com.liferay.project.templates.ProjectTemplateFilesTest$1.preVisitDirectory(ProjectTemplateFilesTest.java:870)
        at com.liferay.project.templates.ProjectTemplateFilesTest$1.preVisitDirectory(ProjectTemplateFilesTest.java:859)
```

Reply:
Yes, I can, @gamerson . But, what should be the correct syntax of the `if`?